addons/hr_expense/tests/test_expenses.py

```diff
@@ -8,6 +8,60 @@
 @tagged('-at_install', 'post_install')
 class TestExpenses(TestExpenseCommon):
 
+    def _get_payment(self, expense_sheet, amount):
+        ctx = {'active_model': 'account.move', 'active_ids': expense_sheet.account_move_id.ids}
+        payment_register = self.env['account.payment.register'].with_context(**ctx).create({
+            'amount': amount,
+            'journal_id': self.company_data['default_journal_bank'].id,
+            'payment_method_id': self.env.ref('account.account_payment_method_manual_in').id,
+        })
+        payment = payment_register._create_payments()
+        return payment
+
+    def test_expense_sheet_payment_state(self):
+        expense_sheet = self.env['hr.expense.sheet'].create({
+            'name': 'Expense for John Smith',
+            'employee_id': self.expense_employee.id,
+            'accounting_date': '2021-01-01',
+            'expense_line_ids': [(0, 0, {
+                'name': 'Car Travel Expenses',
+                'employee_id': self.expense_employee.id,
+                'product_id': self.product_a.id,
+                'unit_amount': 350.00,
+            })]
+        })
+
+        expense_sheet.action_submit_sheet()
+        expense_sheet.approve_expense_sheets()
+        expense_sheet.action_sheet_move_create()
+
+        payment = self._get_payment(expense_sheet, 100.0)
```

Comment:
Why isn't `_get_payment` an inner function since its usage in only restricted for this test?